dist/docker/redhat/Dockerfile

```diff
@@ -5,8 +5,8 @@ MAINTAINER Avi Kivity <avi@cloudius-systems.com>
 ENV container docker
 
 # The SCYLLA_REPO_URL argument specifies the URL to the RPM repository this Docker image uses to install Scylla. The default value is the Scylla's unstable RPM repository, which contains the daily build.
-ARG SCYLLA_REPO_URL=http://downloads.scylladb.com/rpm/unstable/centos/master/latest/scylla.repo
-ARG VERSION=666.development
+ARG SCYLLA_REPO_URL=http://downloads.scylladb.com/rpm/unstable/centos/branch-4.0/latest/scylla.repo
+ARG VERSION=scylla
```

Comment:
If you set `VERSION` to `4.0.*`, we will always install the _latest_ from the unstable repository, which seems like a reasonable default to me.

The release pipelines will pass the stable repository URL and a specific URL.

Reply:
Great. Thanks @penberg